### ISSUE TITLE
bb#60 Takläggarn porter v.3 nov 15

### ISSUE THREAD

**Issue by Bbbolaget-karl** [sweco-seukni/Bbbolaget#45]
ok. 
inklippt från förra
62% pale ale 4,3 kg
21 % vienna 1,5 kg
2 % svartmalt 0,1 kg
4 % chokladmalt 0,3 kg
5% special B. 0,35 kg
6 % crystal 100 0,55 kg

Maltnotan kan nog med fördel ändras. Jag tror vi vinner på att dra ner mängden Vienna då den bara konverterar sig själv och lite lite till. Vi behöver mer kraft här, gissar jag. 
Vidare räknar vi inte in svartmalten i mängden (även om den är liten) då den kallmäskas och kommer i, först i koket. VI kommer också behöva få upp OGn något. Kan tänka mig lite muscovadosocker, ev. 

Vad säger du om notan, U? 

följande inklippt:
70 IBU amarillo @ 60
Mäska vid 65 istället för 68.
Förra hade vi OG 1098 och FG est 1018. Vi nådde FG 1024. Tror på bättre lösning nu. US-05 och högre förjäsbarhet och bättre konverteringsförmåga.

**Comment by Bbbolaget-karl** [comment]
För att sikta vidare på FG est 1018 så kör vi alltå 2 pkt rehydrerad us-05. Vidare kan det vara en idé att ha i lite socker, för utjäsning, eller? 

70 g amarillo @ 60. 
mäska 65
preboil 1074
OG 1078

Hur når vi 1078? tror vi kan nå 1070 med en timmas kok, sen då? 
%satserna är det som gäller avseende Malten. jag räknar med att vi inte har mer än 5,5 kg i pipan. 
90 minuters mäsk?

**Comment by Bbbolaget-karl** [comment]
vi har ljus kristallmalt, om 35 EBC. kan vara i del ersättning för vienna, och resten pale ale då.

**Comment by sweco-seukni** [comment]
Räknar med att du menar 1078 i första inlägget. Är det för att vi har mer vätska som det blir 1070? Det är väl egentligen bara fördelar med att koka längre, isf känns det som bästa sättet att höja OG, vi borde ha tid och kan tappa parallelt. Halvtimmes längre kok och sen muskovadosocker om det behövs?

Vienna byts ut mot 10% ljus karmell och 11% Pale Ale?

Kallmäskningen ger lite sämre utbyte så vi kan ta 130g svartmalt i en halv liter rumstempererat vatten. Svartmalten ska malas så fint som möjligt för bättre utbyte, vissa kör kaffekvarn och silar genom melitta.

**Comment by Bbbolaget-karl** [comment]
Ja, jag tror också det är det som menas. Men det är saxat från tidigare. 

du har rätt vi kokar längre. Säg 90 min är bra. 

Rimligt med malten också. Kanske inte har riktigt så mycket ljus karamell, men jag kan kolla imorgon i affärn också. 

Hur länge skulle kallmäskningen stå? Kaffekvarnen går inte då den är full av kaffe, svår att göra ren. Men jag ser till att mortla. Det är inte så stora mängder. så blir det silning sen också.

**Comment by Bbbolaget-karl** [comment]
Skaffat mer ljus karamellmalt.

**Comment by Bbbolaget-karl** [comment]
Har tittat i receptboken. För robust porter är det rätt måttliga mängder vienna/munich. Men i baltic porter, iofsig m lagerjäst,  är det närmre 35% vienna. Är väl därifrån vi fått idén. Så vi kan göra lite som vi vill. 

Men, den vienna vi har, den har stått i pappåse länge, och är bortåt 7 kg. Tror den har dragit till sig fukt. Därför också skeptisk till att använda den. 

Nu har vi bortåt 800 g ljus karamellmalt

**Comment by sweco-seukni** [comment]
Någon påstod att 4 timmar räcker för kallmäskningen men "över natten" verkar vara allmän praxis. Man kan skaka om blandningen också vid jämna mellanrum för bättre utbytet, men det känns inte nödvändigt.

Något mer som behöver beslutas? Köpte du mer Amarillo?

**Comment by Bbbolaget-karl** [comment]
Maltnotan isf
Vi hade 100 g amarillo
Har köpt us05
Den 20 nov 2015 16:38 skrev "sweco-seukni" <notifications@github.com>:

**Comment by Bbbolaget-karl** [comment]
Faktisk maltnota

76% pale ale 4,3 kg
9 % crystal 35, 0,5 kg
4 % chokladmalt 0,23 kg
5% special B. 0,28 kg
6 % crystal 100 0,34 kg
= 5,70 kg
Därutöver 

Svartmalt. 0,13 kg
kallmäskas. Över natten och över dagen efter.

**Comment by Bbbolaget-karl** [comment]
i senaste brew your own avråder de bestämt från att använda malt som dragit åt sig fukt. 
alltså, borde vi kasta Viennamalten.

**Comment by Bbbolaget-karl** [comment]
130 g svartmalt (weyermann 1200 ebc) 3 ggr genom mandelmjölskvarnen.

**Comment by Bbbolaget-karl** [comment]
130 g svartmalt (weyermann 1200 ebc) 3 ggr genom mandelmjölskvarnen.

**Comment by Bbbolaget-karl** [comment]
Preboil 1055

**Comment by Bbbolaget-karl** [comment]
Tillsätter 360 g kung markatta rårörsocker

**Comment by Bbbolaget-karl** [comment]
Og innan socker 1065

**Comment by Bbbolaget-karl** [comment]
Og 1072

**Comment by Bbbolaget-karl** [comment]
70 g amarillo @ 60 min. 8,5 % alfasyra.  -->70 IBU

**Comment by Bbbolaget-karl** [comment]
Förbi högkröjs. Kopplar ur kylskåpet för att låta jäshinken stå i sin egna värme och i någon mån rampa upp/bibehålla jästemp. 18 gr i omgivande rum.

**Comment by Bbbolaget-karl** [comment]
fg 1017
